src/data_provider/tests/sysInfoPackagesMAC/sysInfoMacPackages_test.cpp

```diff
@@ -158,7 +158,7 @@ TEST_F(SysInfoMacPackagesTest, macPortsValidDataEmptyFields)
 
     EXPECT_EQ(macportsMock.name(), "neovim");
     // Empty string fields are replaced with space.
-    EXPECT_EQ(macportsMock.version(), " ");
+    EXPECT_EQ(macportsMock.version(), "");
```

Comment:
The comment does not reflect the change.

Reply:
fixed